development.md

```diff
@@ -39,6 +39,8 @@ You can use our nightly builds to test new features of upcoming versions. They a
 
 Nightly builds can cause all kinds of crashes and malfunction. Please report such bugs, but don't expect us to deliver a fix right afterwards. If you need stable functionality, use the normal build.
 
+The "light" version is without MLKit translator. it is therefore smaller and without a non FOSS (https://wikipedia.org/wiki/Free_and_open-source_software) part.
```

Comment:
Upper case for new sentence, "FOSS" should be linked directly, "without a non FOSS part"? Overall we should decide how to call it and what's the focus of the build. Is it a FOSS version or a light version?